plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/FilesTable.java

```diff
@@ -65,6 +65,8 @@
     private final TypeManager typeManager;
     private final Table icebergTable;
     private final Optional<Long> snapshotId;
+    private final List<io.trino.spi.type.Type> types;
```

Comment:
why do we care exactly?

Reply:
No strong reason. Just for avoiding redundant computation.